app/db/models.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
```

Comment:
A note about requires:
```javascript
// this first line requires the entire 'mongoose' module
const mongoose = require('mongoose');

// this variable declaration uses destructuring to extract the 'Schema' export from 'mongoose'
const { Schema } = mongoose;
const marketSchema = new Schema({ ... });

// but then this directly calls the 'model' function from 'mongoose'
const Market = mongoose.model('Market', marketSchema);
```
First, [an explanation of what "destructuring" is](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring). It's a very handy concept.

Like a lot of our files, `mongoose` exports its public classes, functions, etc. ([here](https://github.com/Automattic/mongoose/blob/master/index.js)'s where, with the same `module.exports` syntax we use). With object destructuring, we can just extract a single (or a few) exports to require precisely what we need and then use those extracted exports. Or, we can use standard JS notation (since `module.exports` is JSON) to directly access the export that we need.

The above snippet uses both ways of requiring things from the same module, so just be consistent and pick one, i.e.
```javascript
// either
const { model, Schema } = require('mongoose'); // or maybe switched bc 'S' < 'm' in ASCII
const marketSchema = new Schema({ ... });
const Market = model('Market', marketSchema);

// or
const mongoose = require('mongoose');
const marketSchema = new mongoose.Schema({ ... });
const Market = mongoose.model('Market', marketSchema);
```

And then there's always the [`export`](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export) and [`import`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) keywords...